File: 0210.py
class Solution:
    def plusOne(self, digits):
        return [int(digit) for digit in str(int(''.join(map(str, digits))) + 1)]

# Example usage:
solution = Solution()
print(solution.plusOne([1, 2, 3]))   # [1, 2, 4]
print(solution.plusOne([4, 3, 2, 1])) # [4, 3, 2, 2]
print(solution.plusOne([9]))          # [1, 0]

File: 0210.rb
def plus_one(digits)
  (digits.join('').to_i + 1).to_s.chars.map(&:to_i)
end

# Example usage:
puts plus_one([1, 2, 3])   # [1, 2, 4]
puts plus_one([4, 3, 2, 1]) # [4, 3, 2, 2]
puts plus_one([9])          # [1, 0]
